src/lib/network/network_autoyast.rb

```diff
@@ -227,6 +227,27 @@ def valid_rename_udev_rule?(rule)
       true
     end
 
+    def rename_lan_item(item, name_to, attr = nil, key = nil)
+      return if item.nil? || item.empty? || item < 0 || item >= LanItems.Items.size
```

Comment:
please document type of item in documentation. as empty and comparison with 0 looks strange as I have no clue how it can work unless it is own special object:

```
jreidinger@linux-vvcf:~> irb
irb(main):001:0> 0.empty?
NoMethodError: undefined method `empty?' for 0:Fixnum
	from (irb):1
	from /usr/bin/irb:11:in `<main>'
irb(main):002:0> "1" < 0
ArgumentError: comparison of String with 0 failed
	from (irb):2:in `<'
	from (irb):2
	from /usr/bin/irb:11:in `<main>'
irb(main):003:0> [1] < 1
NoMethodError: undefined method `<' for [1]:Array
	from (irb):3
	from /usr/bin/irb:11:in `<main>'
```